internal/plugin/grpc_provider.go

```diff
@@ -81,6 +82,7 @@ func (p *GRPCProvider) GetProviderSchema() (resp providers.GetProviderSchemaResp
 	// check the global cache
 	if !p.Addr.IsZero() {
 		if resp, ok := providers.SchemaCache.Get(p.Addr); ok {
+			logger.Trace("GRPCProvider: GetProviderSchema: serving from global schema cache")
```

Comment:
Wouldn't it be useful to include `p.Addr` in the log entry?

Reply:
Good point, will do!

Reply:
Good idea!